content/en/synthetics/guide/app-that-requires-login.md

```diff
@@ -52,7 +52,9 @@ An alternative would be to use a non-SSO approach and leverage a regular usernam
 
 ### Multi-factor authentication
 
-Browser tests can reproduce any actions a regular user can take inside their browser. If you perform the multi-factor (or 2FA, or TFA) authentication step inside of a browser, you can record it when setting up your browser test. Some MFA providers might however detect Datadog's browser tests as bots and prevent them from logging in, for example, by adding a reCAPTCHA. If that is your case, consider reaching out to your MFA provider to see if it is possible to turn off bot detection when [identifying requests as coming from Synthetic browser tests][3] (for example, for a specific set of credentials, Synthetic tests specific headers, etc.) for testing purposes.
+Datadog Synthetic Monitoring fully [supports Time-based One Time Passwords (TOTPs)][7], a Multi-Factor Authentication method consisting in generating one-time passwords based on combining a secret key and the time.
```

Comment:
```suggestion
Datadog Synthetic Monitoring supports [Time-based One Time Passwords (TOTPs)][7], a multi-factor authentication method that combines a secret key and a set amount of time to generate a one-time password.
```

Reply:
I'm not sure about 'a set amount of time'. TOTP generates a code from a secret key and the time (current day, hour). Maybe using 'timestamp' is more accurate ? or 'Unix time'? 'Current time'?